schemas/data.schema.json

```diff
@@ -57,212 +114,39 @@
-                  "baseline_high_date": {
-                    "description": "Date the feature achieved Baseline high status",
-                    "type": "string"
-                  },
-                  "baseline_low_date": {
-                    "description": "Date the feature achieved Baseline low status",
-                    "type": "string"
-                  },
-                  "support": {
-                    "additionalProperties": false,
-                    "description": "Browser versions that most-recently introduced the feature",
-                    "properties": {
-                      "chrome": {
-                        "type": "string"
-                      },
-                      "chrome_android": {
-                        "type": "string"
-                      },
-                      "edge": {
-                        "type": "string"
-                      },
-                      "firefox": {
-                        "type": "string"
-                      },
-                      "firefox_android": {
-                        "type": "string"
-                      },
-                      "safari": {
-                        "type": "string"
-                      },
-                      "safari_ios": {
-                        "type": "string"
-                      }
-                    },
-                    "type": "object"
-                  }
-                },
-                "required": [
-                  "baseline",
-                  "support"
-                ],
-                "type": "object"
-              },
-              "description": "Statuses for each key in the feature's compat_features list, if applicable. Not available to the npm release of web-features.",
-              "type": "object"
-            },
-            "support": {
-              "additionalProperties": false,
-              "description": "Browser versions that most-recently introduced the feature",
-              "properties": {
-                "chrome": {
-                  "type": "string"
-                },
-                "chrome_android": {
-                  "type": "string"
-                },
-                "edge": {
-                  "type": "string"
-                },
-                "firefox": {
-                  "type": "string"
-                },
-                "firefox_android": {
-                  "type": "string"
-                },
-                "safari": {
-                  "type": "string"
-                },
-                "safari_ios": {
-                  "type": "string"
-                }
-              },
-              "type": "object"
-            }
-          },
-          "required": [
-            "baseline",
-            "support"
-          ],
-          "type": "object"
+          "description": "Whether a feature is considered a \"Baseline\" web platform feature and when it achieved that status",
+          "$ref": "#/definitions/StatusHeadline"
```

Comment:
Similar comment here. baseline and support  were previously required. But I think they should stay required. (Probably in StatusHeadline)

Reply:
Fixed in b2b6762.